src/main/java/io/papermc/hangar/service/OrgFactory.java

```diff
@@ -6,48 +6,78 @@
 import io.papermc.hangar.db.dao.UserDao;
 import io.papermc.hangar.db.model.OrganizationsTable;
 import io.papermc.hangar.db.model.UsersTable;
+import io.papermc.hangar.model.NotificationType;
 import io.papermc.hangar.model.Role;
 import io.papermc.hangar.model.viewhelpers.UserData;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.papermc.hangar.service.sso.AuthUser;
+import io.papermc.hangar.util.HangarException;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
 import org.springframework.stereotype.Component;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestTemplate;
 
 import java.util.Map;
 
 @Component
 public class OrgFactory {
 
     private final HangarDao<OrganizationDao> organizationDao;
+    private final HangarDao<UserDao> userDao;
     private final HangarConfig hangarConfig;
     private final UserService userService;
     private final RoleService roleService;
-
-    /*TEMP*/
-    private final HangarDao<UserDao> userDao;
-    /*END TEMP*/
+    private final NotificationService notificationService;
+    private final RestTemplate restTemplate;
+    private final ObjectMapper mapper;
 
     @Autowired
-    public OrgFactory(HangarDao<OrganizationDao> organizationDao, HangarConfig hangarConfig, UserService userService, RoleService roleService, HangarDao<UserDao> userDao) {
+    public OrgFactory(HangarDao<OrganizationDao> organizationDao, HangarDao<UserDao> userDao, HangarConfig hangarConfig, UserService userService, RoleService roleService, NotificationService notificationService, RestTemplate restTemplate, ObjectMapper mapper) {
         this.organizationDao = organizationDao;
+        this.userDao = userDao;
         this.hangarConfig = hangarConfig;
         this.userService = userService;
         this.roleService = roleService;
-        this.userDao = userDao;
+        this.notificationService = notificationService;
+        this.restTemplate = restTemplate;
+        this.mapper = mapper;
     }
 
     public OrganizationsTable createOrganization(String name, long ownerId, Map<Long, Role> members) {
+        // TODO logging
         String dummyEmail = name.replaceAll("[^a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]", "") + '@' + hangarConfig.org.getDummyEmailDomain();
-        // TODO create dummy user w/email
-        /* TEMPORARY TODO REMOVE */
-        UsersTable usersTable = userDao.get().insert(new UsersTable(name, name, dummyEmail, "I'm an organization", new int[]{ }, false, "en_US"));
-        /* END TEMPORARY */
-        OrganizationsTable org = new OrganizationsTable(name, ownerId, usersTable.getId()); // TODO userId will be from result of dummy user creation
+        HttpHeaders headers = new HttpHeaders();
```

Comment:
We shouldn't attempt to call the auth API if auth is disabled (`fake-user: true`).